app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -105,7 +116,19 @@ protected String requestToken() throws UnableToSendLabOrderException {
         return token;
     }
 
+    protected String retrieveAzureKey(String senderPrivateKey) throws SecretRetrievalException {
```

Comment:
I would remove the need for this variable to be passed in and instead construct that string in this method just like it is outside this method.

`var senderPrivateKey = "report-stream-sender-private-key-" + ApplicationContext.getEnvironment();`